app/views/event_instances/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h3><%= link_to @event_instance.event.name, @event_instance.event %> <%= @event_instance.year %></h3>
+
+<h5>Proposals</h5>
+<%= render partial: 'shared/instance_submissions', locals: { instance: @event_instance } %>
```

Comment:
So, I've just learnt that partials have performance implications on Rails sites — for the worse! Interesting, eh? I'm sure it's not a problem for us now, but just interesting to know.